device/device.yaml

```diff
@@ -17,21 +17,35 @@ components:
       x-include: 
       - '../common/common.yaml#/components/schemas/Named.Object'
       description: >-
-        A container for emulated interfaces and protocol devices.
+        A container for emulated interfaces, loopback interfaces and protocol configurations.
       type: object
-      required: [container_name, ethernet]
       properties:
-        container_name: 
+        ethernets:
           description: >-
-            The unique name of a Port or Lag object that will contain the 
-            emulated interfaces and/or protocol devices.
-          type: string
-          x-constraint:
-          - '/components/schemas/Port/properties/name'
-          - '/components/schemas/Lag/properties/name'
-        ethernet:
+            The emulated Ethernet settings.
+          type: array
+          items:
+            $ref: './ethernet.yaml#/components/schemas/Device.Ethernet'
+        ipv4_loopbacks:
           description: >-
-            The ethernet stack.
-          $ref: './ethernet.yaml#/components/schemas/Device.Ethernet'
-    
-  
+            IPv4 Loopback interface settings attached to an Ethernet interface.
+          type: array
+          items:
+            $ref: './ipv4.yaml#/components/schemas/Device.Ipv4Loopback'
+        ipv6_loopbacks:
+          description: >-
+            IPv6 Loopback interface settings attached to an Ethernet interface.
+          type: array
+          items:
+            $ref: './ipv6.yaml#/components/schemas/Device.Ipv6Loopback'      
```

Comment:
@ajbalogh had mentioned this earlier which I agree with: `ipv4_loopbacks` and `ipv6_loopbacks` should be defined inside `ethernets`.

I read an answer from @apratimmukherjee but didn't understand why this can't be addressed with the aforementioned approach.
> The problem with this is for chained Devices where there will be only Loopbacks attached to a Ethernet on another connected Device. Hence, we probably do need a separate Loopback interface with reference to the Ethernet it is chained behind.

Does this imply e.g. ipv4 loopback `lo1` defined inside `d1` can be bound to `eth2` ethernet device defined in `d2` ? If so, could we clearly document following constraints ?
- can d1 and d2 have different port names ?
- can ipv4/v6 loopbacks and routers be bound to interfaces defined in other devices ?

Reply:
"Does this imply e.g. ipv4 loopback lo1 defined inside d1 can be bound to eth2 ethernet device defined in d2 ?" --> Yes.
"can d1 and d2 have different port names ?" -->No. A device does not have port name but the the ethernet has.
"can ipv4/v6 loopbacks and routers be bound to interfaces defined in other devices" --> yes.
Please let know the follow desciption is ok or not:
IPv4 Loopback interface settings attached to an Ethernet interface.
A loopback address is defined under one device can be bound to an Ethernet
that is defined under another device.